query-engine/core/src/metrics/mod.rs

```diff
@@ -95,9 +105,14 @@ fn describe_metrics() {
     absolute_counter!("query_total_queries", 0);
     absolute_counter!("query_total_operations", 0);
 }
+
+static METRIC_RECORDER: Once = Once::new();
```

Comment:
This is not a problem with multiple loaded nodeAPI instances? I remember a previous comment on the larger metrics PR that this is merely collecting and fanning out metrics or something, just making sure it won't interfere if you instantiate multiple nodeAPI prisma clients.

Reply:
It is needed for multiple nodeApi instances because each one will try and load the metric recorder. It only ever needs to be loaded once. So if a second instance tries to load it, it panics.